src/test/codegen/consts.rs

```diff
@@ -43,14 +43,14 @@ pub fn inline_enum_const() -> E<i8, i16> {
 #[no_mangle]
 pub fn low_align_const() -> E<i16, [i16; 3]> {
     // Check that low_align_const and high_align_const use the same constant
-    // CHECK: load %"E<i16, [i16; 3]>"*, %"E<i16, [i16; 3]>"** bitcast (<{ i8*, [0 x i8] }>* [[LOW_HIGH]] to %"E<i16, [i16; 3]>"**),
+    // CHECK: load %"E<i16, [i16; 3]>"*, %"E<i16, [i16; 3]>"** bitcast (<{ i8*, [0 x i8] }>* @1 to %"E<i16, [i16; 3]>"**), align 8, !nonnull !{{[0-9]}}
```

Comment:
Since the part of the test that binds `[[LOW_HIGH]]` is still here, we should also keep this, or replace `[[LOW_HIGH:@[0-9]+]]` with `@2`.

I don't understand why this is still unnamed. Maybe `static_addr_of` is used in more places?

Reply:
replaced in 36c4c6a570909129e4c27833a46e60f4a982c43a